skype/skype-ps/skype/Get-CsCallQueue.md

```diff
@@ -0,0 +1,126 @@
+---
+external help file: Get-CsCallQueue.xml
+applicable: Skype for Business Online
+title: Get-CsCallQueue
+schema: 2.0.0
+---
+
+# Get-CsCallQueue
+
+## SYNOPSIS
+The Get-CsCallQueue cmdlet returns the identified Call Queues.
+
+## SYNTAX
+
+```
+Get-CsCallQueue [-BypassDualWrite <Object>] [-DomainController <Object>] [-Force] [-Identity <Object>]
+ [-Tenant <Object>] [-AsJob] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Get-CsCallQueue cmdlet lets you retrieve information about the Call Queues in your organization. Call Queue output contains statistical data on the number of active calls that are in the queue.
+
+If called without parameters, Get-CsCallQueue returns a collection of all Call Queues configured to be used in your organization.
+
+The Get-CsCallQueue cmdlet may suggest additional steps required to complete the Call Queue setup.
+
+## EXAMPLES
+
+### -------------------------- Example 1 --------------------------
+```
+Get-CsCallQueue
+```
+
+This example gets all Call Queues in the organization.
+
+### -------------------------- Example 2 --------------------------
+```
+Get-CsCallQueue -Identity 5e3a575e-1faa-49ff-83c2-5cf1c36c0e01
+```
+
+This example gets the Call Queue with the identity 5e3a575e-1faa-49ff-83c2-5cf1c36c0e01. If no Call Queue exists with the identity 5e3a575e-1faa-49ff-83c2-5cf1c36c0e01, then this example generates an error.
+
+
+## PARAMETERS
+
+### -DomainController
+PARAMVALUE: Fqdn
+
+```yaml
+Type: Object
+Parameter Sets: (All)
+Aliases: DC
+Applicable: Skype for Business Online
```

Comment:
trying to understand this field, is it required to mention "Skype For Business Online"

Reply:
this was reused from the huntgroup cmdlet markdown files